iHome/ihome.frontend/src/app/pages/rooms/device/rgbcontrol/rgbcontrol.component.scss

```diff
@@ -14,7 +14,7 @@
     justify-content: center;
     align-items: center;
     flex-direction: column;
-    height: 9rem !important;
+    height: 5rem !important;
```

Comment:
!Important 😮 
Rozumiem że bez tego nie chciało działać?

Reply:
Tak, bo material z góry to narzuca nawet jak napiszesz w klasie